### ISSUE TITLE
Some hitboxes are hidden when they're relevant

### ISSUE THREAD

**Issue by wererogue** [struz/smash-move-viewer#2]
See https://struz.github.io/smash-move-viewer/#/v1/sonic/AttackAirLw/19?showAllMoves=false&speed=1
According to the move data, Sonic's down air has a large hitbox on frame 19 that only hits grounded opponents, and a small hitbox that only hits aerial opponents.  Because the large hitbox's ID is lower the visualization hides the small hitbox, assuming it will be overridden by the large one - but in actuality the small hitbox can connect, since the ground-only hitbox won't hit an aerial opponent.

tl;dr Draw order should consider properties which allow an overlapped hitbox to connect.
